modules/firehose/config.go

```diff
@@ -22,23 +25,58 @@ var (
 
 	//go:embed schema/scale.json
 	scaleActionSchema string
+
+	//go:embed schema/reset.json
+	resetActionSchema string
 )
 
 type moduleConfig struct {
-	State          string             `json:"state"`
-	ReleaseConfigs helm.ReleaseConfig `json:"release_configs"`
+	State           string `json:"state"`
+	ChartVersion    string `json:"chart_version"`
+	FirehoseConfigs struct {
```

Comment:
this can just be `Firehose` (and `firehose` in json). Since it's already enclosed inside the config struct, the extra `Configs` suffix feels redundant.

Reply:
Done!